src/components/CommentList.jsx

```diff
@@ -0,0 +1,33 @@
+import CommentCard from "./CommentCard";
+import { useEffect, useState } from "react";
+import { getCommentsForReview } from "./API-calls";
+
+const CommentList = ({ review_id }) => {
+  const [commentList, setCommentList] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  useEffect(() => {
+    setIsLoading(true);
+    getCommentsForReview(review_id).then(({ comments }) => {
+      setCommentList(comments);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) {
+    return <h1>Loading...</h1>;
+  } else {
+    return (
+      <>
```

Comment:
It might be nice from a user experience point of view to give some messaging when there aren't any comments for a review. this is optional, but a nice touch!

Reply:
Ah yeah, I'll add that before merging :)